commands.js

```diff
@@ -526,6 +526,7 @@ exports.commands = {
 				this.addModCommand("" + user.name + " turned on modjoin.");
 			} else if (target in Config.groups) {
 				if (room.battle && !this.can('makeroom')) return;
+				if (room.isPersonal && !this.can('makeroom') && target !== '+') return this.errorReply("/modjoin - Access denied from setting modjoin past + Personal Rooms.");
```

Comment:
`this.can` gives an error message. If you want to replace that error message, you need to use `user.can`